x-pack/plugin/ent-search/src/test/java/org/elasticsearch/xpack/entsearch/engine/EngineIndexServiceTests.java

```diff
@@ -114,35 +111,31 @@ public void testListEngine() throws Exception {
         }
 
         {
-            SearchResponse resp = awaitListEngine("*:*", 0, 10);
-            assertNotNull(resp.getHits());
-            assertThat(resp.getHits().getHits().length, equalTo(10));
-            assertThat(resp.getHits().getTotalHits().value, equalTo(10L));
+            Tuple<Engine[], Integer> searchResponse = awaitListEngine("*:*", 0, 10);
+            final Engine[] engines = searchResponse.v1();
+            assertNotNull(engines);
+            assertThat(engines.length, equalTo(10));
```

Comment:
I wonder if this notation is the preferred Elasticsearch convention, rather than using the shorter `assertEquals(10, engines.length)`.

Reply:
good call! I guess this is more readable in terms of expected / actual value. Maybe Jim has some comments here on styling.

Reply:
I also find the hamcrest version more readable too, it also gives better error message. I don't think we have a strict convention here though.